lgrcommands/render.py

```diff
@@ -0,0 +1,60 @@
+"""
+
+"""
+import re
+import urllib.parse
+
+from pycldf import orm
+import jinja2
+
+from cldfbench_lgr import Dataset
+
+MD_LINK_PATTERN = re.compile(r'\[(?P<label>[^]]+)]\((?P<url>[^)]+)\)')
+
+
+def render_template(d, fname_or_component, obj, index=False, fmt='md'):
+    templateLoader = jinja2.FileSystemLoader(searchpath=str(d))
+    templateEnv = jinja2.Environment(loader=templateLoader, trim_blocks=True)
+    template = templateEnv.get_template('{}_{}.{}'.format(
+        fname_or_component, 'index' if index else 'detail', fmt))
+    return template.render(ctx=obj)
```

Comment:
@fmatter so, using Jinja2's `FileSystemLoader`, we can now use a hierarchical lookup of templates - thereby providing a way to override defaults. As with `clld`, templates are looked up based on
- object type
- index or details view
- output format